storage/kafkasql/src/main/java/io/apicurio/registry/storage/impl/kafkasql/sql/KafkaSqlSink.java

```diff
@@ -58,6 +58,13 @@
      * @param record
      */
     public void processMessage(ConsumerRecord<MessageKey, MessageValue> record) {
+        // If the value is null, then this is a tombstone (or unrecognized) message and should not 
+        // be processed.
+        if (record.value() == null) {
+            log.info("Discarded a tombstone message with key: {}", record.key());
```

Comment:
what about logging some info from the MessageKey?

Reply:
Oh right, I intended to implement `toString()` on the message key classes.  Will do that.

Reply:
Updated to give each key a unique `toString()`.